Makefile

```diff
@@ -108,6 +108,49 @@ generate-charts: build ## Re-generate the helm chart testdata and docs samples
 	(cd docs/book/src/cronjob-tutorial/testdata/project && ../../../../../../bin/kubebuilder edit --plugins=helm/v1-alpha)
 	(cd docs/book/src/multiversion-tutorial/testdata/project && ../../../../../../bin/kubebuilder edit --plugins=helm/v1-alpha)
 
+.PHONY: update-demo
```

Comment:
Could it not be simplified?
It seems too much 

I think I would have expected it more like it is done in the operator-controller project ,for example, see: https://github.com/operator-framework/operator-controller/blob/main/Makefile#L492-L499

Reply:
Thanks a lot @camilamacedo86 the comment is addressed, and the Makefile target is now slim as in the operator controller